content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -11,26 +11,43 @@ Google Cloud.
 
 This guide assumes the following settings:
 
+- The `${MGMT_PROJECT}`, `${MGMT_DIR}` and `${MGMT_NAME}` environment variables
+  are the same as in [Management cluster setup](./management-setup#environmental-variables).
```

Comment:
@joeliedtke what do you think about this approach?
The env variables are too common, I think duplication will be too much efforts to maintain (also hard for users to be sure they are exactly the same among all the documentation pages).
So I'd prefer defining them once when they occur for the first time, and always provide a reference at other docs that use them.

What do you think?

Reply:
To reuse an environment var in multiple docs, I would recommend creating shortcodes that create the environment variable and define what it contains. Each environment var should be defined on the page that it is used on. Users should not have to refer to a different page to understand the environment variables that are used in the code samples.